libs/core/activity/IActivitySequencer.cpp

```diff
@@ -261,4 +261,28 @@ data::ActivitySeries::sptr IActivitySequencer::getActivity(
 
 //------------------------------------------------------------------------------
 
+void IActivitySequencer::clearLastActivities(const data::SeriesDB::sptr& seriesDB, size_t index)
```

Comment:
I'm a bit worried with the "clear" word: Usually, clearing means removing everything to pristine state. I would prefer removeLastActivities.